presto-cassandra/pom.xml

```diff
@@ -185,6 +185,16 @@
         </dependency>
     </dependencies>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.slf4j</groupId>
```

Comment:
Where is this bring used?

Reply:
Yes @yingsu00 .

 slf4j comes as a transitive dependency in org.testcontainers. If we do not add this dependencymanagement tag
the following error will come. 
```
Rule 0: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps.
.....
.....

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce (default) on project presto-cassandra: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. 

```

Inorder to fix this I have added the slf4j version as  <dependencyManagement> tag.

Reply:
@yingsu00  Added the comment.